docs/lib/storage/fragments/js/copy.md

```diff
@@ -0,0 +1,116 @@
+## Copy
+
+`Storage.copy` method copies an existing file to a different key.
+
+It returns a `{key: new S3 Object key}` object on success:
+
+<amplify-callout>
```

Comment:
I think in order of this to work, it needs a blank line between after `<amplify-callout>` and before `</amplify-callout>`